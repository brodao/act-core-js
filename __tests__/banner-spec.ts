import log from '../src/logger';

test("Apresentação do 'Splash'", () => {
	expect(
		log.showBanner({
			name: 'test_show_banner',
			version: '99.99.99',
			description: 'Show Banner',
		})
	).toMatchSnapshot();
});
