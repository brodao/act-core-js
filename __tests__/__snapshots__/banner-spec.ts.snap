// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apresentação do 'Splash' 1`] = `undefined`;
